robot/impl/local_robot.go

```diff
@@ -461,20 +462,20 @@ func newWithResources(
 		}
 	}()
 
-	if cfg.Cloud != nil && cfg.Cloud.AppAddress != "" {
-		var err error
-		r.cloudConn, err = config.CreateNewGRPCClient(ctx, cfg.Cloud, logger)
-		if err != nil {
-			return nil, err
-		}
-		r.packageManager, err = packages.NewCloudManager(pb.NewPackageServiceClient(r.cloudConn), cfg.PackagePath, logger)
-		if err != nil {
-			return nil, err
-		}
-	} else {
-		r.logger.Debug("Using no-op PackageManager when Cloud config is not available")
-		r.packageManager = packages.NewNoopManager()
-	}
+	// if cfg.Cloud != nil && cfg.Cloud.AppAddress != "" {
+	// 	var err error
+	// 	r.cloudConn, err = config.CreateNewGRPCClient(ctx, cfg.Cloud, logger)
```

Comment:
I'm open to disagreement, but my usual preference is not to comment out code: just delete it entirely. Commented-out code tends to rot over time as things change around it. If we need this code again, it's in the git history, or someone can view this PR on github again.

[Example koan, from a website where every anecdote involves someone getting killed and that's not the interesting part of the story so don't get too distracted by that part](http://thecodelesscode.com/case/41)

Reply:
i generally agree - trying to see if there's a better way to do this. if not, we are probably going to make an additional comment that is linked to a ticket to revert these lines in a way that works.